interface/src/app/mlmodel/resource.js

```diff
@@ -0,0 +1,9 @@
+angular.module('adage.mlmodel.resource', ['ngResource'])
+
+.factory('MLModel', ['$resource', function($resource) {
```

Comment:
@mhuyck and @dongbohu : Following the logic of the naming of the other variables and components in this PR, do you think the name of this resource should change to ``MlModel``?